### DISCUSSION TITLE
Right clicking the osu!lazer icon dragged from start menu to taskbar doesn't open context menu

### DISCUSSION THREAD

**Discussion by tuszmak** [ppy/osu#22918] [Q&A]
### Type

Other

### Bug description

The icon that the game opens DOES have context menu, the one which I use to start the game does not.

Excuse me for the windows language. Switching it to english makes the windows really slow somehow. I tried doing it slow so you can navigate with the icons.

### Screenshots or videos

https://user-images.githubusercontent.com/81630417/226121937-15032870-acc7-4992-8994-929a38f942ec.mp4

### Version

2023.305.0-lazer

### Logs

[performance.log](https://github.com/ppy/osu/files/11009135/performance.log)
[runtime.log](https://github.com/ppy/osu/files/11009136/runtime.log)
[updater.log](https://github.com/ppy/osu/files/11009137/updater.log)
[database.log](https://github.com/ppy/osu/files/11009138/database.log)
[input.log](https://github.com/ppy/osu/files/11009139/input.log)
[network.log](https://github.com/ppy/osu/files/11009140/network.log)

**Comment by peppy** [comment]
See https://github.com/ppy/osu/issues/10588.

You might fare better if you pin to the taskbar after running the app, rather than from the start menu. Or it may be due to you having an older installation that is still in the broken state.
